virt/virtual_machines/virt-accessing-vm-ssh.adoc

```diff
@@ -0,0 +1,84 @@
+:_content-type: ASSEMBLY
+[id="virt-accessing-vm-ssh"]
+= Using SSH to access virtual machines
+include::_attributes/common-attributes.adoc[]
+:context: virt-accessing-vm-ssh
+
+toc::[]
+
+You can configure an SSH connection to access virtual machines (VMs) by using the following methods:
+
+* `virtctl ssh` command
++
+You add a public SSH key to a VM before it boots for the first time. Then, you connect to the VM by using the `virtctl ssh` command.
++
+This method is useful for troubleshooting a VM because it requires little time to set up. The connection is slow, but it works consistently even if you are behind a firewall or if the cluster does not have a load balancer.
++
+[NOTE]
+====
+Only the user who created the SSH key pair can use the `virtctl ssh` command.
+====
+
+// dynamic key injection - CNV-30068
+// OpenSSH and virtctl - CNV-26737
+
+* Kubernetes service
++
+You create a service to expose a VM port, add the service to the VM, and then connect by using the `ssh` command.
++
+This method is more time-consuming to set up but it provides better performance than `virtctl`.
+
```

Comment:
Are seconday networks (layer2, SRIOV) skipped by intention?

Reply:
Not intentional. I will see where I can add them.

Reply:
@dominikholler Just to make sure we are talking about the same thing, see  "Connecting using a layer 2 interface" near the end of John Herr's [blog](https://cloud.redhat.com/blog/connecting-to-vms-in-red-hat-openshift-virtualization). Is that what you are referring to? Even with the security risks?

Reply:
yes, this is covered by
https://docs.openshift.com/container-platform/4.13/virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.html
https://docs.openshift.com/container-platform/4.13/virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.html